### ISSUE TITLE
Make us able to remove locked wrenches from drawers

### ISSUE THREAD

**Issue by altmines** [IntegerLimit/StorageDrawers-1.12.2-dev#1]
**Is your feature request related to a problem? Please describe.**
When i put a wrench into the drawer by accident, and took it out, it was locked at that time so i tried unlocking it, putting in the wrench and taking it out again. However, it instead just rotates the drawer.

**Describe the solution you'd like**
Make shift right clicking the drawer with a gt crowbar remove the item locking maybe? so if u accidently put in another item u can just wrench out the filter.

**Describe how the solution will work**
Describe how the solution will fix the problem, or add to the pack. 
It would be a nice qol and make manage drawers easier.

**Describe alternatives you've considered**
NIL

**Additional context**
Add any other context or screenshots about the feature request here. If you are suggesting a mod, please also add a link to the mod's CurseForge page.

**Comment by IntegerLimit** [comment]
Oh, I think I'll do something like using a key unlocks + removes item display if drawer has no items.

I'm just going to add here so I don't forget: make wrenches only rotate drawers, so you can't put wrenches inside drawers.

**Comment by IntegerLimit** [comment]
Gonna Transfer this to our new storage-drawers fork.

**Comment by a-a-GiTHuB-a-a** [comment]
What about the case where someone _wants_ a wrench drawer?

**Comment by IntegerLimit** [comment]
> What about the case where someone _wants_ a wrench drawer?

Any ideas?

**Comment by IntegerLimit** [comment]
Oh they can use conduits 💯

**Comment by a-a-GiTHuB-a-a** [comment]
Oh yeah, conduits. Do item pipes work as well?

**Comment by IntegerLimit** [comment]
Yeah, think so. Any ideas about an exception to allow people to do it manually though? Or not really needed?

**Comment by a-a-GiTHuB-a-a** [comment]
I like the idea of using `Shift+RMB` the most.

**Comment by a-a-GiTHuB-a-a** [comment]
I mean, is it meant to do something else? I don't think it's used anywhere else, so maybe Shift+RMB can be used for items that interact with drawers like wrenches and keys.
